lib/jellyfish_web/api_spec/component/file.ex

```diff
@@ -13,7 +13,14 @@ defmodule JellyfishWeb.ApiSpec.Component.File do
       title: "ComponentPropertiesFile",
       description: "Properties specific to the File component",
       type: :object,
-      properties: %{}
+      properties: %{
+        filePath: %Schema{
+          type: :string,
+          description: "Path to track file. Must be either OPUS encapsulated in Ogg or raw h264",
+          example: "/root/video.h264"
```

Comment:
Isn't `filePath` a related path? Because `/root/video.h264`, which appears to be an absolute path, could easily be misunderstood."